akka-projection-slick/src/main/scala/akka/projection/slick/SlickProjection.scala

```diff
@@ -124,6 +123,9 @@ object SlickProjection {
   /**
    * Create a [[akka.projection.Projection]] with at-least-once processing semantics.
    *
+   * The DBIO returned by the [[SlickHandler.process()]] of the provided [[handler]] will be
+   * wrapped in a transaction.
+   *
```

Comment:
Here is makes more sense. Because users need to know that if they do: `dbio.flatMap(_=> anotherDbio)` the two will be on the same transaction.